designer/client/components/Process/types.ts

```diff
@@ -79,7 +79,7 @@ export type StatusType = {
 export type StatusDefinitionType = {
   name: string,
   displayableName: string,
-  icon?: string,
-  tooltip?: string,
+  icon: string,
+  tooltip: string,
```

Comment:
What about `ProcessStateType`?
and places of usages like `processState?.tooltip || unknownTooltip()` ?

Reply:
Fixed `ProcessStateType`. The second example handles empty status, theoretically it is possible (see `BaseProcessDetails`) but is should not happen in real life.